ci/docker/run/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get install -y git build-essential  autoconf flex bison \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Delete the apt-get lists after installing something](https://app.codacy.com/manual/wsnyder/verilator/pullRequest?prid=4756540)